Resources/Prototypes/Catalog/Fills/Boxes/ammunition.yml

```diff
@@ -351,110 +351,117 @@
 
 # base BallisticAmmoProvider boxes
 - type: entity
-  parent: BoxMagazine
+  parent: BaseItem
   id: BoxAmmoProvider
   abstract: true
   components:
-  - type: BallisticAmmoProvider
   - type: ContainerContainer
     containers:
       ballistic-ammo: !type:Container
       storagebase: !type:Container
+  - type: Item
+    size: 30
+  - type: Sprite
+    sprite: Objects/Storage/boxes.rsi
+
+- type: entity
+  parent: BoxAmmoProvider
+  id: BoxAmmoProviderShotgunShell
+  abstract: true
+  components:
+    - type: BallisticAmmoProvider
+      whitelist:
+        tags:
+        - ShellShotgun
+      capacity: 12
```

Comment:
Looks like this still has a container attached to it?

Reply:
My bad, I missed that because I made the extra container "vestigial" so it was not noticeable or interactable in testing. As I understand the one named ballistic-ammo is still required, for the shells? Like how all the firearm magazines have it?